x/avs/types/expected_keepers.go

```diff
@@ -42,6 +44,8 @@ type OperatorKeeper interface {
 	GetOptedInOperatorListByAVS(ctx sdk.Context, avsAddr string) ([]string, error)
 	GetOperatorOptedUSDValue(ctx sdk.Context, avsAddr, operatorAddr string) (operatortypes.OperatorOptedUSDValue, error)
 	GetAVSUSDValue(ctx sdk.Context, avsAddr string) (sdkmath.LegacyDec, error)
+	SetOperatorInfo(ctx sdk.Context, addr string, info *operatortypes.OperatorInfo) (err error)
+	QueryOperatorInfo(ctx context.Context, req *operatortypes.GetOperatorInfoReq) (*operatortypes.OperatorInfo, error)
```

Comment:
Similar question here: why not use `keeper.OperatorInfo(` directly?

Reply:
fixed